auto-exporters/zipkin/src/main/java/io/opentelemetry/auto/exporters/zipkin/ZipkinExporterFactory.java

```diff
@@ -23,19 +23,12 @@
 import zipkin2.reporter.okhttp3.OkHttpSender;
 
 public class ZipkinExporterFactory implements SpanExporterFactory {
-  private static final String ZIPKIN_ENDPOINT = "zipkin.endpoint";
-  private static final String DEFAULT_ZIPKIN_ENDPOINT = "http://localhost:9411/api/v2/spans";
-
-  private static final String ZIPKIN_SERVICE_NAME = "zipkin.service.name";
-  private static final String DEFAULT_ZIPKIN_SERVICE_NAME = "(unknown service)";
 
   @Override
   public SpanExporter fromConfig(Config config) {
-    final String zipkinEndpoint = config.getString(ZIPKIN_ENDPOINT, DEFAULT_ZIPKIN_ENDPOINT);
-    final String serviceName = config.getString(ZIPKIN_SERVICE_NAME, DEFAULT_ZIPKIN_SERVICE_NAME);
     return ZipkinSpanExporter.newBuilder()
-        .setSender(OkHttpSender.create(zipkinEndpoint))
```

Comment:
I forgot about this, I think we need to force `OkHttpSender` here due to https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/375#issuecomment-630452595

Reply:
ok, trying to set it.

Reply:
Anyways except for when the JAR size is extremely limited, Zipkin always recommends using OkHttp in production since performance is way better.

Reply:
I opened https://github.com/open-telemetry/opentelemetry-java/issues/1409 to change the default upstream, so we don't have to override it in the future